src/components/links/link/link.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import Styled from '@emotion/styled';
+import vars from '../../../../materials/custom-properties';
+
+const createStyledComponent = component => Styled(component)`
+  font-family: ${vars.fontFamily};
+  color: ${vars.colorPrimary};
+  font-size: ${vars.fontSizeDefault};
+
+  &:hover {
+    color: ${vars.colorPrimary25};
+  }
+`;
+
+const StyledReactRouterLink = createStyledComponent(ReactRouterLink);
+const StyledExternalLink = createStyledComponent('a');
+
+const componentProps = ['isExternal', 'to'];
+
+const Link = props => {
+  // instead of object destructing
+  const remainingProps = Object.keys(props)
+    .filter(key => !componentProps.includes(key))
+    .reduce((obj, key) => {
+      // eslint-disable-next-line no-param-reassign
+      obj[key] = props[key];
+      return obj;
+    }, {});
```

Comment:
Should this instead maybe be a general helper? Sounds a bit like a pattern which will start to be repeated. Like `getPassThroughProps`.